users/jdelkins_ss

```diff
@@ -0,0 +1 @@
+jdelkins
```

Comment:
Is this file needed? 
If not, it should be removed.

Reply:
This is a symlink to my userspace. It is needed unless there is another straightforward way to include my `jdelkins` userspace from my `jdelkins_ss` keymaps?

Reply:
Missed that in the comments.  And yes, there is, actually. 
https://docs.qmk.fm/#/feature_userspace?id=override-default-userspace

Specifically, if you add this to the rules.mk for the layouts:
```makefile
USER_NAME := jdelkins
```

This overrides the userspace that is used.

Reply:
That's fantastic. Thanks!